charts/nginx-gateway-fabric/values.yaml

```diff
@@ -477,3 +477,32 @@ certGenerator:
 
   # -- Overwrite existing TLS Secrets on startup.
   overwrite: false
+
+# Example gateway object:
```

Comment:
nit: I would put this below the `gateways` field. Makes more sense to me from a flow perspective.